MultiServer.py

```diff
@@ -1641,6 +1641,7 @@ async def process_client_cmd(ctx: Context, client: Client, args: dict):
         elif cmd == 'LocationScouts':
             locs = []
             create_as_hint: int = int(args.get("create_as_hint", 0))
+            slot: int = int(args.get("player", client.slot))
```

Comment:
this seems to be a new argument that needs documenting in the network api

Reply:
occurs to me this would return a LocationInfo response with locations potentially belonging to other players, with no way for the client to differentiate, not sure how to best handle that